enterprise_scheduler/executor.py

```diff
@@ -100,9 +100,15 @@ def execute_task(self, task):
 
             result = client.post('/models', **files)
 
-            print("Training URL : http://{}:{}/#/trainings/{}/show".format(urlparse(config.api_endpoint).netloc.split(":")[0],
-                                                                      ffdl_ui_port,
-                                                                      result['model_id']))
+            if 'models' not in result:
```

Comment:
I believe this solves the specific error case. But, in general, does result have any indication of error so we can make a more generic error handling and on the case of errors, provide a little more informative message to the user ?

Reply:
I'm not sure how FDDL handles errors since from our testing I only came across 2 cases where result is returned without an exception (exception handling was already in the code) but is still an error case. In the case of a malformed request, result returns with a `message` detailing the error, which is what this handles, displaying the message. the second case was where return contains `models` as expected but it's empty. This seems to indicate that the request was successfully submitted but the job failed to start on the FFDL side (based on what I understood from when you guys were debugging Alan's instance. Any error messaging in that case would need to be added FFDL side.